x-pack/test_serverless/api_integration/test_suites/common/index_management/datastreams.ts

```diff
@@ -21,10 +21,8 @@ export default function ({ getService }: FtrProviderContext) {
   let internalReqHeader: InternalRequestHeader;
   const svlDatastreamsHelpers = getService('svlDatastreamsHelpers');
 
-  describe('Data streams', function () {
-    // see details: https://github.com/elastic/kibana/issues/187372
-    this.tags(['failsOnMKI']);
-
+  // see details: https://github.com/elastic/kibana/issues/187372
+  describe.skip('Data streams', function () {
```

Comment:
Can you please help me understand this change? This linked issue only mentions failures when running on MKI but the `.skip` also disables is for local runs. Also, it looks like you "moved skip tests up to a skipped suite", but there's a nested sub-suite that was not skipped before and now is. Just wondering if this all is intended. Maybe just the linked issue needs an update?